strings/tips.txt

```diff
@@ -230,4 +229,4 @@ You can swap floor tiles by holding a crowbar in one hand and a stack of tiles i
 When hacking doors, cutting and mending the "test light wire" will restore power to the door.
```

Comment:
cutting and mending _a_ test light wire. There are two wires that turn off the test light